packages/playwright-test/package.json

```diff
@@ -33,6 +33,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@types/node": "*",
+    "chokidar": "3.5.3",
```

Comment:
I think instead we want to say: ^3.5 to not break the ecosystem when other tools like vite also already depend on it.

Reply:
I am actually thinking the opposite, I'd put it into bundle if I could.